### PR TITLE
Upgrading the payment service to micronaut 4

### PR DESCRIPTION
**Pull request by ChaimaaeROUAI**

### CODE REVIEW

File: src/payment/app/build.gradle [hunk 1]
```diff
@@ -1,69 +1,92 @@
...
+dockerBuild {
+    images = ["phx.ocir.io/oraclelabs/micronaut-showcase/mushop/$project.parent.name-$project.name-${javaBaseImage}:$project.version"]
+}
+
+dockerBuildNative {
+    images = ["phx.ocir.io/oraclelabs/micronaut-showcase/mushop/${project.parent.name}-${project.name}-native:$project.version"]
+}
+
+dockerfileNative {
+    instruction "RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.11/main' >> /etc/apk/repositories && apk update && apk add 'zlib<1.2.12'"
```
Comment (alvarosanchez) [inline src/payment/app/build.gradle:91]: I don't think this is needed today, can you check?

Comment (ChaimaaeROUAI) [reply src/payment/app/build.gradle:91]: yes it works without this

Comment (alvarosanchez) [reply src/payment/app/build.gradle]: Then please remove all occurrences

Comment (ChaimaaeROUAI) [reply src/payment/app/build.gradle]: Done

**Review by alvarosanchez — APPROVED** [review]

**Review by OthmanMilahOracle — APPROVED** [review]

**Review by Julia3072 — APPROVED** [review]

**Pull request merged**